mteb/models/voyage_models.py

```diff
@@ -95,8 +95,7 @@ def encode(
         **kwargs: Any,
     ) -> np.ndarray:
         input_type = (
-            self.get_prompt_name(self.model_prompts, task_name, prompt_type)
-            or "document"
+            self.get_prompt(self.model_prompts, task_name, prompt_type) or "document"
         )
```

Comment:
It's known from `self.model_prompts` that voyage models only accept "query" and "document" as input types. Maybe this can be further simplified to the following? That way, the `get_prompt` method is not needed.
```suggestion
input_type = self.model_prompts.get(prompt_type.value)
```